test/test_helper.rb

```diff
@@ -32,6 +32,27 @@
   ActiveRecord::Migration.execute "SET CONSTRAINTS ALL DEFERRED;"
 end
 
+# List of valid record name field.
+def valid_name_list
```

Comment:
I think this is generally a good idea, could you extract to separate PR with also changing existing tests to use it? Speaking about https://github.com/theforeman/foreman/blob/develop/test/models/organization_test.rb#L8-L28 unless there's a reason we can't use it there.

Reply:
@ares this is a more general function that should be valid for the most entities that's why I putted here,  unlike the org_name_list that is valid only for organization as it has it's own limits.

Reply:
Ok, makes sense